nucypher/network/nodes.py

```diff
@@ -1359,7 +1359,7 @@ def node_details(node):
                    "last_seen": last_seen,
                    "fleet_state": node.fleet_state_checksum or 'unknown',
                    "fleet_state_icon": fleet_icon,
-                   "domain": node.learning_domain,
+                   "domain": node.serving_domain,
```

Comment:
This will help with the monitor - the other problematic line was https://github.com/nucypher/nucypher/blob/ad6b6bda51a548f1b5592a3a7050a7c92d8bbb10/nucypher/network/nodes.py#L353.

Reply:
My current workaround for the monitor - https://github.com/derekpierre/nucypher/commit/ad2e2d0524d90d368c5fb90f71c1c1bf3b46b10a

Reply:
That's line is tricky, because I know that it doesn't fail in other contexts (e.g., when it's a file-based node storage). Not sure how to proceed here until #2356 is merged. What about an ugly try block?

Reply:
I'm good with the ugly for now - with a todo to fix properly later. Just gets us moving before a more comprehensive solution.

Reply:
What do you think of this for line 353?
```
            try:  # Workaround until #2356 is fixed
                node_domain = node.domain.decode('utf-8')
            except:
                node_domain = node.serving_domain
```

Reply:
Looks good. We can be specific about the `AttributeError` - but looks good to me anyway.

Reply:
I'm not sure if it's an attribute error or a BytestringSplitter error. In any case, it's ugly af and we need to fix #2356 soon 😅

Reply:
In a follow-up PR we can collapse `learning_domain` and `serving_domain` into `domain`.